benchmark/package.json

```diff
@@ -4,12 +4,13 @@
   "dependencies": {
```

Comment:
Maybe doesn't totally fit the mold - but It might be nice to move this to `packages/` in the future

Reply:
Yeah, might make sense.